tests/Unit/DockerComposeReconciliationDeploymentTest.php

```
@@ -0,0 +1,161 @@
+<?php
+
+use App\Jobs\ApplicationDeploymentJob;
+use Illuminate\Support\Collection;
+
+function dockerComposeDeploymentMethodSource(): string
+{
+    $source = file_get_contents(__DIR__.'/../../app/Jobs/ApplicationDeploymentJob.php');
+    $start = strpos($source, 'private function deploy_docker_compose_buildpack(): void');
```

PHPStan (2.2.8)

[warning] Parameter `#1` $haystack of function strpos expects string, string|false given.


(argument.type)